### PR TITLE
31040 header background logo

### PR DESCRIPTION
**Pull request by HierenFeb**
Add header background.

Add the logo and visually-hidden site name.

Add positioning for logo, search.

Please review it

https://agov_pr486.agov.com.au/styleguide/agov_whitlam/section-components.html#kssref-components-header

### CODE REVIEW

File: themes/agov/agov_whitlam/sass/_variables.scss [hunk 1]
```diff
@@ -133,6 +133,7 @@ $chroma: add-colors((
   black:                  #000,
 
   blue:                   #0072b9,
+  blue-drark:             #466bc8,
   blue-steel:             #466bc8,
```
Comment (JohnAlbin) [inline themes/agov/agov_whitlam/sass/_variables.scss]: This is a duplicate color.

File: themes/agov/agov_whitlam/sass/sections/_header.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.header {
+  background: color(header-background);
+}
```
Comment (JohnAlbin) [inline themes/agov/agov_whitlam/sass/sections/_header.scss]: The "header" is still a component. Even if its not re-used. Please put in the components folder.

**Comment by HierenFeb** [comment]
@JohnAlbin  I updated, please review it! Thanks!

**Comment by JohnAlbin** [comment]
Reviewing…

File: themes/agov/agov_whitlam/sass/components/header/_header.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
+// Header
+//
+// The header of page.
+//
+// @Work
+//
+// Style guide: components.header
```
Comment (JohnAlbin) [inline themes/agov/agov_whitlam/sass/components/header/_header.scss]: This is not done.

File: themes/agov/agov_whitlam/templates/page.tpl.php [hunk 4]
```diff
@@ -11,43 +11,43 @@
 <div class="page">
```
Comment (JohnAlbin) [inline themes/agov/agov_whitlam/templates/page.tpl.php]: Remove the `<div class="page">` element. It was needed with Zen's layout, but not with this agov design.

File: themes/agov/agov_whitlam/templates/page.tpl.php [hunk 5]
```diff
@@ -11,43 +11,43 @@
...
 
   <header class="header" role="banner">
+    <div class="header__inner clearfix">
+      <?php if ($logo): ?>
+        <a href="<?php print $front_page; ?>" title="<?php print t('Home'); ?>" rel="home" class="header__logo"><img src="<?php print $logo; ?>" alt="<?php print t('Home'); ?>" class="header__logo-image" /></a>
+      <?php endif; ?>
```
Comment (JohnAlbin) [inline themes/agov/agov_whitlam/templates/page.tpl.php]: The `<div class="header__inner clearfix">` element should be using the `layout-center` class.

File: themes/agov/agov_whitlam/sass/components/header/_header.scss [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+.header__inner {
+  max-width: $container;
+  margin: 0 auto;
+  padding: 30px 35px 20px;
+
+  @include respond-to(medium) {
+    padding-left: 10px;
+    padding-right: 10px;
+  }
+}
```
Comment (JohnAlbin) [inline themes/agov/agov_whitlam/sass/components/header/_header.scss]: Most of these properties of `.header__inner` are all duplicates of `.layout-center`, aren't they? I think the padding top/bottom is all that is needed.

File: themes/agov/agov_whitlam/templates/page.tpl.php [hunk 7]
```diff
@@ -47,74 +45,72 @@
...
+            'level' => 'h2',
+            'class' => array('visually-hidden'),
+          ),
+        )); ?>
+      </nav>
     <?php endif; ?>
 
+    <?php print render($page['navigation']); ?>
+
   </div>
```
Comment (JohnAlbin) [inline themes/agov/agov_whitlam/templates/page.tpl.php]: The navigation region will need to be moved to just below the header, but we can do that in a different PR.

Comment (HierenFeb) [inline themes/agov/agov_whitlam/templates/page.tpl.php]: ok @JohnAlbin, we will do that later

**Comment by JohnAlbin** [comment]
Looks good! Merging.
